MIGRATING.md

```diff
@@ -125,6 +125,14 @@ major releases of `cosmwasm`. Note that you can also view the
   +CosmosMsg::Any(AnyMsg { type_url, value })
   ```
 
+- Replace all direct construction of `StdError` with use of the corresponding
+  constructor:
+
+  ```diff
+  -StdError::GenericErr { msg }
+  +StdError::generic_err(msg)
+  ```
```

Comment:
Thanks. Could you also add a note on the removed feature? We already have a block for that in line 22

Reply:
Good catch!